data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -2056,7 +2056,7 @@
         [message]
             speaker=Abhai
             # wmllint: local spelling er
-            message= _ "Wouldn’t miss it. Maybe after he is destroyed my kind and I can live... er... be dead peacefully."
+            message= _ "Wouldn’t miss it. Maybe after he is destroyed me and my kind can live... er... be dead peacefully."
```

Comment:
I actually preferred the formal version for Abhai, since he's the ghost of a king/ruler.